slither/tools/read_storage/read_storage.py

```diff
@@ -119,7 +127,32 @@ def get_storage_layout(self) -> None:
                 elif isinstance(type_, ArrayType):
                     elems = self._all_array_slots(var, contract, type_, info.slot)
                     tmp[var.name].elems = elems
-
+        for contract, var in self.constant_slots:
+            var_name = var.name
+            try:
+                exp = var.expression
+                if str(exp).startswith("0x"):
+                    slot = coerce_type("int", str(exp))
+                elif str(exp).startswith("keccak256(bytes)"):
+                    slot_str = str(exp).split("(")[2].replace(")", "")
+                    slot_hash = keccak(text=slot_str)
+                    slot = coerce_type("int", slot_hash)
```

Comment:
> Which line(s) are you referring to?
I think you can get the literal value from the expression here

Reply:
Got it, made the change